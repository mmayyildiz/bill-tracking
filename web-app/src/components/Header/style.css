.header{
    background: #00b3f5;
    text-align: center;
    padding: 8px;
    color: #fff;
    font-size:20px;
}

.sub-header{
    text-align: center;
    margin: auto;
    width: 40%;
}

.sub-header-title{
    color: #445368;
    font-size: 22px;
}

.sub-header-title-description{
    color: #8d99aa;
}