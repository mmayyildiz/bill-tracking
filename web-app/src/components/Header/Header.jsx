import React from "react";
import './style.css'

const Header = ({title, subTitle, description}) => {

   return (
     <React.Fragment>
        <div className="header">
            <p>  {title} </p>
        </div>
        <div className="sub-header">
          <p className="sub-header-title"> {subTitle} </p>
          <p className="sub-header-title-description"> {description}
          </p>
       </div>
   </React.Fragment>
   )

}

export default Header;


