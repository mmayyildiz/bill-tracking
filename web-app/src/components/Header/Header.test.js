import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

const props = {
    title: 'foo title',
    subTitle: 'foo sub',
    description: 'foo desc'
 }

 const mountedComponent = shallow(<Header {...props}/>);

describe('Header', () => {

    it('should render with props', () => {
       
        expect(mountedComponent).toMatchSnapshot();
    });

    it('should render paragraphs', () => {
        expect(mountedComponent.find('p').length).toBe(3);
    });

});
