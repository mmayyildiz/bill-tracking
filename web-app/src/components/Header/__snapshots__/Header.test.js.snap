// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render with props 1`] = `ShallowWrapper {}`;
